src/helpers/getHoverClassName.ts

```diff
@@ -0,0 +1,15 @@
+import { AdaptivityProps } from "../hoc/withAdaptivity";
+
+export function getHoverClassName(
+  base: string,
+  hover?: AdaptivityProps["deviceHasHover"]
+): string {
+  let hoverState = "none";
+  if (hover === true) {
+    hoverState = "has";
+  } else if (hover === false) {
+    hoverState = "has-not";
```

Comment:
судя по `.Tappable--hover-none` должно быть

```suggestion
    hoverState = "none";
```

Reply:
`none` это значение когда `hover === undefined`

Reply:
а, лол, точно)) не заметил, сорре